dd-sdk-android/src/test/kotlin/com/datadog/android/DatadogTest.kt

```diff
@@ -227,6 +231,51 @@ internal class DatadogTest {
         verify(CoreFeature.trackingConsentProvider).setConsent(fakeConsent)
     }
 
+    @Test
+    fun `M no changes W initialize() { verboseDebug } in non debug mode`() {
```

Comment:
Can you add `// Given`, `// When` and `// Then` comments to split your tests in 3 parts?